docroot/modules/custom/sitenow_media_wysiwyg/config/schema/sitenow_media_wysiwyg.schema.yml

```diff
@@ -0,0 +1,13 @@
+media_entity_static_map.settings:
+  type: config_object
+  label: 'Media entity static map settings'
+  mapping:
+    link:
+      type: uri
+      label: 'Link'
+    zoom:
+      type: integer
+      label: 'Zoom'
+    label:
```

Comment:
It might've been pulled from a different media entity. At the time I thought it was needed to get the fields to appear. It clearly wasn't in the end, I'll remove it.

Reply:
i think you need to remove the entire file if empty for travis to be happy

Reply:
I thought it already existed but looking back I apparently did add it, removing now.